### PR TITLE
Update release notes for 19.2

### PR DESCRIPTION
**Pull request by geriux**
This PR updates an entry from the release notes to specify that the new feature for font size/line height is only available for [block themes](https://developer.wordpress.org/block-editor/how-to-guides/themes/block-theme-overview/).

From @khaykov This PR also updates an entry from the release notes to specify that the new comment moderation feature in Reader post comments is only for removal of comments (not approval).

To test: N/A

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding unit tests for my changes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/jetpack_metadata/release_notes.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- See text blocks in the block editor adjust with changing font size and line height.
+- Font size and line height support for block themes.
```
Comment (AliSoftware) [inline WordPress/jetpack_metadata/release_notes.txt]: 🎉  That makes the notes even shorter now, going from 354 to 322 characters!

Even leaves enough room to make that a bit more sentence-y in the "long" notes like here if you want (while keeping the concise wording for the "short" alternative)?  🤷  Up to you, really, I'm fine with both.
```suggestion
- Font size and line height are now supported for block themes.
```
```suggestion
- Block themes now support font size and line height adjustments.
```

**Review by AliSoftware — COMMENTED** [review]
Looking good, and makes the "long" release notes even shorter (and under our 350 character limit goal) 👏  Even gave some suggestions for alternative wordings, take it or leave it.

---

@ParaskP7 I'll let you the responsibility to take care of running the lane to update the `.po` file according (I think it's fine for you to checkout that PR's branch and run the lane directly on it to add that change to that PR, but if you prefer to do it in a separate PR on top of this one that works for me as well). Once you've done it, the Peril warning should also magically disappear.

**Comment by ParaskP7** [comment]
👋 @AliSoftware @geriux !

> @ParaskP7 I'll let you the responsibility to take care of running the lane to update the .po file according (I think it's fine for you to checkout that PR's branch and run the lane directly on it to add that change to that PR, but if you prefer to do it in a separate PR on top of this one that works for me as well). Once you've done it, the Peril warning should also magically disappear.

I am on it! ⚙️

**Comment by ParaskP7** [comment]
👋 @AliSoftware @geriux !

> I am on it! ⚙️

This is now done! ✅

File: WordPress/jetpack_metadata/release_notes.txt (hunk 1 above)
Comment (geriux) [reply WordPress/jetpack_metadata/release_notes.txt]: Oh nice! That would be better yes, thanks for the suggestion. I see the metadata strings were updated already. Am I too late? 😅 I'm fine with either option though.

Comment (ParaskP7) [reply WordPress/jetpack_metadata/release_notes.txt]: 👋 @geriux !

Not to worry, just do any additional updates as you feel like and let me (or @AliSoftware) know afterwards. I can re-run the script one more time, it is no big of a deal. 💯

Comment (geriux) [reply WordPress/jetpack_metadata/release_notes.txt]: Thanks! Updated!

Comment (ParaskP7) [reply WordPress/jetpack_metadata/release_notes.txt]: Coolio, I am seeing you update the `long` version of the `Jetpack` release notes only. Is that on purpose @geriux ?

Comment (geriux) [reply WordPress/jetpack_metadata/release_notes.txt]: My bad 😅 , I've just updated the WordPress one too.

Comment (ParaskP7) [reply WordPress/jetpack_metadata/release_notes.txt]: 😅 Great, it is looking good now, thank you! 🌟

**Comment by ParaskP7** [comment]
👋 @khaykov !

I will wait for you to add the `Reader` related release note change as well before I proceed with running the script on my side.

**Comment by khaykov** [comment]
Thanks, @ParaskP7 ! Update the notes for reader (and PR description).

**Review by AliSoftware — APPROVED** [review]
All changes (2 for WP, 1 for JP) looks good 👍

**Pull request merged**